JBang project fails to start using 1.10.0.Final version

**Describe the bug**
JBang project fails to start using 1.10.0.Final version for dependencies version

**Expected behavior**
It should be able to start

**Actual behavior**
Fails to start with the following postBuild error

> [jbang] Resolving dependencies...
[jbang]     Resolving io.quarkus:quarkus-resteasy:1.10.0.Final...Done
[jbang] Dependencies resolved
[jbang] Building jar...
[jbang] Post build with io.quarkus.launcher.JBangIntegration
Warning: failed to load configurator: java.util.ServiceConfigurationError: org.jboss.logmanager.EmbeddedConfigurator: Provider io.quarkus.bootstrap.logging.InitialConfigurator not a subtype
        at java.util.ServiceLoader.fail(ServiceLoader.java:239)
        at java.util.ServiceLoader.access$300(ServiceLoader.java:185)
        at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:376)
        at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404)
        at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
        at org.jboss.logmanager.LogContext.getConfigurator(LogContext.java:222)
        at org.jboss.logmanager.LogContext.<clinit>(LogContext.java:38)
        at org.slf4j.impl.Slf4jLoggerFactory.getLogger(Slf4jLoggerFactory.java:34)
        at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:363)
        at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:388)
        at org.eclipse.aether.internal.impl.DefaultRepositoryEventDispatcher.<clinit>(DefaultRepositoryEventDispatcher.java:45)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.newInstance(DefaultServiceLocator.java:164)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.getInstances(DefaultServiceLocator.java:139)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.getInstance(DefaultServiceLocator.java:125)
        at org.eclipse.aether.impl.DefaultServiceLocator.getService(DefaultServiceLocator.java:278)
        at org.eclipse.aether.internal.impl.DefaultMetadataResolver.initService(DefaultMetadataResolver.java:119)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.newInstance(DefaultServiceLocator.java:169)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.getInstances(DefaultServiceLocator.java:139)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.getInstance(DefaultServiceLocator.java:125)
        at org.eclipse.aether.impl.DefaultServiceLocator.getService(DefaultServiceLocator.java:278)
        at org.apache.maven.repository.internal.DefaultVersionResolver.initService(DefaultVersionResolver.java:108)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.newInstance(DefaultServiceLocator.java:169)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.getInstances(DefaultServiceLocator.java:139)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.getInstance(DefaultServiceLocator.java:125)
        at org.eclipse.aether.impl.DefaultServiceLocator.getService(DefaultServiceLocator.java:278)
        at org.eclipse.aether.internal.impl.DefaultRepositorySystem.initService(DefaultRepositorySystem.java:143)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.newInstance(DefaultServiceLocator.java:169)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.getInstances(DefaultServiceLocator.java:139)
        at org.eclipse.aether.impl.DefaultServiceLocator$Entry.getInstance(DefaultServiceLocator.java:125)
        at org.eclipse.aether.impl.DefaultServiceLocator.getService(DefaultServiceLocator.java:278)
        at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.newRepositorySystem(BootstrapMavenContext.java:662)
        at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.getRepositorySystem(BootstrapMavenContext.java:223)
        at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.resolveRemoteRepos(BootstrapMavenContext.java:476)
        at io.quarkus.bootstrap.resolver.maven.BootstrapMavenContext.getRemoteRepositories(BootstrapMavenContext.java:231)
        at io.quarkus.bootstrap.JBangBuilderImpl.postBuild(JBangBuilderImpl.java:27)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at io.quarkus.launcher.JBangIntegration.postBuild(JBangIntegration.java:77)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at dev.jbang.IntegrationManager.runIntegration(IntegrationManager.java:94)
        at dev.jbang.cli.BaseBuildCommand.buildJar(BaseBuildCommand.java:206)
        at dev.jbang.cli.BaseBuildCommand.build(BaseBuildCommand.java:129)
        at dev.jbang.cli.Run.prepareArtifacts(Run.java:70)
        at dev.jbang.cli.Run.doCall(Run.java:59)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:80)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:12)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
        at picocli.CommandLine.access$1200(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
        at picocli.CommandLine.execute(CommandLine.java:2058)
        at dev.jbang.Main.main(Main.java:14)
Nov 27, 2020 9:25:43 AM io.quarkus.deployment.QuarkusAugmentor run
WARN: Using Java versions older than 11 to build Quarkus applications is deprecated and will be disallowed in a future release!
[jbang] [ERROR] Issue running postBuild()
[jbang] Run with --verbose for more details


**To Reproduce**

Steps to reproduce the behavior:
1. Generate a new JBang project using the command `jbang init -t qrest test.java`
2. Edit `test.java` file DEPS area dependency version to 1.10.0.Final version (//DEPS io.quarkus:quarkus-resteasy:1.10.0.Final)
3. Run project by executing `jbang test.java`

--- a/core/launcher/src/main/java/io/quarkus/launcher/JBangIntegration.java
+++ b/core/launcher/src/main/java/io/quarkus/launcher/JBangIntegration.java
@@ -40,7 +40,8 @@ public class JBangIntegration {
 
                         @Override
                         protected Class<?> loadClass(String name, boolean resolve) throws ClassNotFoundException {
-                            if (name.startsWith("org.") && !(name.startsWith("org.xml.") || name.startsWith("org.w3c."))) {
+                            if (name.startsWith("org.") && !(name.startsWith("org.xml.") || name.startsWith("org.w3c.")
+                                    || name.startsWith("org.jboss."))) {
                                 //jbang has some but not all of the maven resolver classes we need on its
                                 //class path. These all start with org. so we filter them out to make sure
                                 //we get a complete class path
@@ -51,7 +52,8 @@ public class JBangIntegration {
 
                         @Override
                         public URL getResource(String name) {
-                            if (name.startsWith("org/") && !(name.startsWith("org/xml/") || name.startsWith("org/w3c/"))) {
+                            if (name.startsWith("org/") && !(name.startsWith("org/xml/") || name.startsWith("org/w3c/")
+                                    || name.startsWith("org/jboss/"))) {
                                 //jbang has some but not all of the maven resolver classes we need on its
                                 //class path. These all start with org. so we filter them out to make sure
                                 //we get a complete class path
@@ -62,7 +64,8 @@ public class JBangIntegration {
 
                         @Override
                         public Enumeration<URL> getResources(String name) throws IOException {
-                            if (name.startsWith("org/") && !(name.startsWith("org/xml/") || name.startsWith("org/w3c/"))) {
+                            if (name.startsWith("org/") && !(name.startsWith("org/xml/") || name.startsWith("org/w3c/")
+                                    || name.startsWith("org/jboss/"))) {
                                 //jbang has some but not all of the maven resolver classes we need on its
                                 //class path. These all start with org. so we filter them out to make sure
                                 //we get a complete class path

